packages/mobile/e2e/init.js

```diff
@@ -2,14 +2,23 @@ const detox = require('detox')
 const config = require('../package.json').detox
 const adapter = require('detox/runners/jest/adapter')
 const specReporter = require('detox/runners/jest/specReporter')
+const assignReporter = require('detox/runners/jest/assignReporter')
 
-jest.setTimeout(120000)
 jasmine.getEnv().addReporter(adapter)
+// This takes care of generating status logs on a per-spec basis. By default, jest only reports at file-level.
+// This is strictly optional.
 jasmine.getEnv().addReporter(specReporter)
 
+// This will post which device has assigned to run a suite, which can be useful in a multiple-worker tests run.
+// This is strictly optional.
+jasmine.getEnv().addReporter(assignReporter)
+
+// Increase default jest timeout
+jest.setTimeout(120000)
+
 beforeAll(async () => {
   await detox.init(config)
-})
+}, 300000)
```

Comment:
Nice addition! We should add a comment as to why this is needed.
From what I saw, jest swallows errors or timeouts here and happily continues with executing the test suite though this crucial init failed.

Reply:
I'm not 100% sure, I'm just aligning this file with the example on Detox's repo